Rearrange Guppy's interior

ghostsheet opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->
:cl: 
tweak: Rearranged the Guppy's interior for quality of life conveniences.
/:cl:

Shuffles around the guppy's interior
To create some quality of life changes with the following map changes:

Makes the cell charger more accessible.
Moves the airlock controller over the rack. (which stops it from being blocked by machinery or drill parts and makes it always accessible).
Moves the sensor console, allowing a passenger to access it while seated (space trips are already boring as is).
Moves engine console, allowing pilot to access it while seated.
Moves helm console, ensures that only the pilot can reach it.
Close the fuel pipe valve and increase its layer so it's on top of the other pipe.
Change SMEs location.
Change airlock sensor location.
Moves light fixture.

![guppy before after](https://user-images.githubusercontent.com/43085828/52158111-169af400-26e9-11e9-8e0d-21f05c144926.png)

ghostsheet committed:
Update torch-0.dmm

SMES shuffle

As per Roland410's request, I've shuffled the guppy again to expose the SMES while retaining all of it's flight control.

ghostsheet committed:
Update torch-0.dmm

Roland410 commented:
Fairly sure you can't use the SMES like that, also please turn the fuel pipe valve back to open.

ghostsheet commented:
The SMES can be accessed through climbing on top the consoles, as it's rarely used anyways. The fuel valve is closed because I wanted it to be open as part of the prep for the guppy (oh the industrial hazard feel) but I can change it, boss

Roland410 commented:
A lot of people turn the SMES output off once they land on a planet or similiar to conserve power (I do it myself as well).

ghostsheet committed:
SMES shuffle

As per Roland410's request, I've shuffled the guppy again to expose the SMES while retaining all of it's flight control.

ghostsheet commented:
Guppy 2.0
![guppy after 3](https://user-images.githubusercontent.com/43085828/52161321-8627d800-2717-11e9-9759-4f6b4c64fbe2.png)

ghostsheet committed:
Merge branch 'Guppy-shuffle' of https://github.com/ghostsheet/Baystation12 into Guppy-shuffle

ghostsheet committed:
Revert "Update torch-0.dmm"

This reverts commit 6921d846b21c12f0ec3549d018259ebbb4a0406c.

ghostsheet commented:
ahhhh running into errors and merge conflict with myself I'mma just brute force it and start a new pr
